bazel/repositories.bzl

```diff
@@ -399,6 +400,25 @@ def _com_google_absl():
         name = "abseil_optional",
         actual = "@com_google_absl//absl/types:optional",
     )
+    native.bind(
+        name = "abseil_synchronization",
+        actual = "@com_google_absl//absl/synchronization:synchronization",
+    )
+    native.bind(
+        name = "abseil_time",
+        actual = "@com_google_absl//absl/time:time",
+    )
+
+def _com_googlesourcecode_cctz():
+    _repository_impl("com_googlesource_code_cctz")
+    native.bind(
+        name = "abseil_time_zone",
+        actual = "@com_googlesource_absl//civil_time",
+    )
+    native.bind(
+        name = "abseil_time_zone",
+        actual = "@com_googlesource_absl//time_zone",
```

Comment:
Does `abseil_time` have a direct dependency requiring `//external:abseil_time_zone` to exist? This seems surprising, most projects (including Envoy) are moving away from using bind. I didn't spot this in https://github.com/abseil/abseil-cpp/blob/master/absl/time/BUILD.bazel.

Reply:
As it happens I just pinged the abseil folks and they claim a recent commit there removes the timezone library dependence.  Will cut the cord & commit an update when I get a decent internet connection.

I don't know much about 'bind' here; was just copying patterns :)